app/views/sessions/new.html.erb

```diff
@@ -1,3 +1,18 @@
-<h1> Login With Google </h1>
-
-<%= link_to 'Login With Google', '/auth/google_oauth2' %>
+<div class='center-search'>
+  <p>Please, login using Google.</p>
+</div>
```

Comment:
Very basic google login link centered in the middle of the page with our color scheme.